src/common/string_util.cc

```diff
@@ -121,7 +121,7 @@ int StringMatch(const std::string &pattern, const std::string &in, int nocase) {
 }
 
 // Glob-style pattern matching.
-int StringMatchLen(const char *pattern, int patternLen, const char *string, int stringLen, int nocase) {
+int StringMatchLen(const char *pattern, size_t patternLen, const char *string, size_t stringLen, int nocase) {
```

Comment:
Since I noticed it via your PR, you can change the camel case to the snake case here for function arguments.

Reply:
Changed